bode/bode/models/task.py

```diff
@@ -9,14 +11,23 @@
 from bode.models.utc_datetime import UTCDateTime
 
 
+class Status(enum.Enum):
```

Comment:
Can we change the name to `TaskStatus`, `Status` is a bit vague word

Reply:
Endorse. Name `Status` is definitely too general.
Also, it'd be more coherent with e.g. TaskRelation.